features/step_definitions/admin_steps.rb

```diff
@@ -70,6 +70,20 @@ def update_settings(settings)
   Language.find_or_create_by_short_and_name("FR", "Francais")
 end
 
+Given /^caching is enabled$/ do
```

Comment:
I was going to use them to further test the Controller but couldn't figure it out, but thought these step defs would be good to keep for future tests. I can pull them out if you'd like. Let me know.